src/alchemlyb/tests/parsing/test_amber.py

```diff
@@ -282,3 +282,13 @@ def test_concatenated_amber_dhdl(tmp_path):
         match="TI Energy detected after the TIMINGS section.",
     ):
         extract(tmp_path / "amber.out", 298)
+
+
+def test_correct_timestamp_u_nk(testfiles):
```

Comment:
@yuxuanzhuang Thanks for adding the test. I think it is fine to just adapt this test to use an existing file and check if the timestep is correct would be enough. No need to add a new file.

Reply:
thanks. then I think `test_u_nk_time_reading(single_u_nk)` is already checking it.

Reply:
Ok, so it was already checked, I was thinking something along the line of 
```
def test_u_nk_time_reading_bar_intervall(single_u_nk, tmp_path):
    """Test if time information is read correctly when extracting u_nk"""
    with bz2.open(single_u_nk, 'rt') as bz2_file:
        content = bz2_file.read()

    with open(tmp_path / 'amber.log', 'w') as text_file:
        text_file.write(content.replace("bar_intervall =     1000", "bar_intervall =     100"))
    u_nk = extract_u_nk(tmp_path / 'amber.log', T=298.0)
    assert_allclose(u_nk.index.values[0][0], ??)
```
Then you show that changing the bar_intervall in the log file will change the time stamp in the generated dataframe.